generate_tags.py

```diff
@@ -0,0 +1,100 @@
+import json
+
+import requests
+
+# This script loads the enterprise-attack.json mitre dataset into a dict (mitre_dataset)
+# according to the parameters set below. The dict is then iterated to produce an attack.py
+# file, containing variables for the multiple Mitre ATT&CK tags relevant to the project.
+#
+# The enterprise-attack.json file is sourced from Mitre's repository at
+# https://github.com/mitre-attack/attack-stix-data/tree/master/enterprise-attack.
+
+# Replace the URL below with the version of your choice.
+
+mitre_source = (
+    "https://raw.githubusercontent.com/mitre-attack/attack-stix-data/"
+    "master/enterprise-attack/enterprise-attack-14.1.json"
+)
+
+# Begin parameters; modify these to customize the tags produced by the script
+
+included_platforms = ["Windows", "Linux"]
+include_subtechniques = True
+include_deprecated = False  # include techniques marked as "deprecated"
+include_revoked = False  # include techniques marked as "revoked"
```

Comment:
For the foreseeable future, we'll always want these options set how they are. Let's remove these options and simplify the code.